src/stylesheets/_variables.module.scss

```diff
@@ -0,0 +1,10 @@
+$mobile: 480px;
+$tablet: 640px;
+$desktop: 1024px;
+
+// division by 1px removes the unit
```

Comment:
These css responsive sizes were exported for use in jsx files, specifically for the custom hook I created to detect mobile changes.  Previously mobile changes were either using css (current global nav - not applicable in the case of the sidenav).  The custom hook is to detect resonsive changes in the code to apply certain components.  Those are exported to have a single constant for our responsive limits

Reply:
@ClayBenson94 Yes, the division means we're exporting `"480"` instead of `"480px"`, which simplifies the parsing `src/utils/checkMobile.tsx` needs to do to get a number.